commands/env.go

```diff
@@ -141,6 +155,9 @@ func loadBackendEnsureUser(env *Env) func(*cobra.Command, []string) error {
 // This wrapper style is necessary because a Cobra PostE function does not run if RunE return an error.
 func closeBackend(env *Env, runE func(cmd *cobra.Command, args []string) error) func(*cobra.Command, []string) error {
 	return func(cmd *cobra.Command, args []string) error {
+		env.err = out{Writer: cmd.ErrOrStderr()}
```

Comment:
This code allows redirection of stderr and stdout during testing.  Since the these streams haven't been redirected by the `cmd` during normal operation, this has no effect for the production behavior.